model/modelindexer/indexer.go

```diff
@@ -442,6 +442,10 @@ func (r *pooledReader) Read(p []byte) (int, error) {
 	return n, err
 }
 
+func (r *pooledReader) Seek(offset int64, whence int) (int64, error) {
```

Comment:
Why is there a Seek method now? Does this mean that on io.EOF, the reader may be reused? (See Read above)

Reply:
I believe this is related to this change https://github.com/elastic/go-elasticsearch/pull/315.

Reply:
I took a look at the code, we only implement it because we're still using the elasticsearch.BulkIndexerItem struct. We have our own BulkIndexer implementation which never calls the Seek method, so we're good.